### DISCUSSION TITLE
Get automatic default window arrangement for N tiles

### DISCUSSION THREAD

**Discussion by rsimon** [nomcopter/react-mosaic#212] [Q&A] [answered]
Hi,

is there a recommended way (or built-in utility?) that would give me a default `initialValue` automatically from a set of IDs?

**Comment by nomcopter** [comment, marked as answer]
The exported function `createBalancedTreeFromLeaves` should do what you want!

**Reply by rsimon** [reply]
Wonderful, thanks!
